### PR TITLE
add docs about creating custom permission

### PR DESCRIPTION
**Pull request by punkle**

### CODE REVIEW

**Review by Irma12 — APPROVED** [review]

File: content/docs/details/custom-plugins/index.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
+---
+title: Using Permissions in Custom Plugins
+publishedDate: '2024-12-18T21:00:00.0Z'
+description: How to configure and manage permissions and policies for use in Custom Plugins in Roadie.
+---
+
+## Introduction
+
+**This feature is currently in the process of being rolled out. If you want access, please contact our support or sales teams.**
```
[the remark this answers is not in the document]
Comment (Ian-Link-Roadie) [reply content/docs/details/custom-plugins/index.md:9]: @punkle :point_up:

[the remark this answers is not in the document]
Comment (punkle) [reply content/docs/details/custom-plugins/index.md:9]: This is already a pattern I am copying from another page.. Id suggest if we are improving this we should make the change across the other pages as part of another commit.

Its a good suggestion though.

[the remark this answers is not in the document]
Comment (Ian-Link-Roadie) [reply content/docs/details/custom-plugins/index.md:9]: It is something I'm trying to move towards as we write new documentation. I have used it in multiple pages already, an example would be on step 18 of the [Track Fragments ](https://roadie.io/docs/tech-insights/track-fragments/) recipe. If you prefer, we can do a bulk update, but I've been doing greenfield page as I create them.

[the remark this answers is not in the document]
Comment (punkle) [reply content/docs/details/custom-plugins/index.md:9]: Brilliant. Im all on for it. Let me know if i can help.

**Pull request merged**